provider/core/src/hello_world.rs

```diff
@@ -41,6 +41,21 @@ impl Default for HelloWorld<'_> {
     }
 }
 
+impl<'a> ZeroFrom<'a, str> for HelloWorld<'a> {
```

Comment:
nb: I still want it on the record that I disapprove of implicitly using `ZeroFrom` to do the reverse transform. It's not much better than `From`, it can just borrow better.

Reply:
#6183